docker-compose.yml

```
@@ -3,101 +3,155 @@
 # Docker compose to build Telemeta meta project
 
 version: '2.1'
+
+networks:
+  keycloak: {}
+  telemeta: {}
+  services:
```

```suggestion
  services:
```
Permet de se connecter avec d'autres service qui ne sont pas présents dans ce  *Docker-compose*